src/ET_Client.php

```diff
@@ -954,7 +954,7 @@ function CreateContentAreas($arrayOfContentAreas)
 	 * @param string $str string to be validated
 	 */
 	function isNullOrEmptyString($str){
-        return (!isset($str) || trim($str) === '');
+        return (!isset($str) || trim($str ?? '') === '');
```

Comment:
!isset(null) will return true and the second part of this will never be executed if it's null, so the change shouldn't be necessary here.